tasklist/client/src/v2/Tasks/AvailableTasks/index.test.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {
+  render,
+  screen,
+  waitForElementToBeRemoved,
+  within,
+} from 'common/testing/testing-library';
+import {AvailableTasks} from './index';
+import {Link, MemoryRouter} from 'react-router-dom';
+import {nodeMockServer} from 'common/testing/nodeMockServer';
+import {http, HttpResponse} from 'msw';
+import * as tasksMocks from 'v1/mocks/tasks';
```

Comment:
Do we plan to move the mocks at some point to v2 or make them common ?

Reply:
It's done on the following PR